### ISSUE TITLE
Reading pdf documents

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2782]
**Reported by aravind on 2012-11-04 06:44**
NVDA is unable to read pdf documents. Can you please help me on this.

**Comment by ehollig** [comment]
I think this issue is invalid, as there is not enough information provided.
